rotkehlchen/accounting/structures/processed_event.py

```diff
@@ -111,6 +111,7 @@ def to_exported_dict(
             exported_dict['cost_basis_taxable'] = taxable_basis
             exported_dict['cost_basis_free'] = free_basis
             exported_dict['asset'] = str(self.asset)
+            exported_dict['asset_symbol'] = self.asset.resolve_to_asset_with_symbol().symbol
```

Comment:
this can raise wrongassettype (irc) if the assets is not one of the types that has symbols

Reply:
Can you ad one entry to the test with `CustomAsset` ?